default/python/AI/freeOrionAIInterface.pyi

```diff
@@ -1973,9 +1981,18 @@ def getTech(string: str) -> tech:
     """
 
 
-def getTechCategories(obj: object) -> StringVec:
+@overload
+def getTechCategories(obj: object)  -> object: ...
+
+
+@overload
+def getTechCategories()  -> object: ...
+
+
+def getTechCategories(*args) -> object:
     """
     Returns the names of all tech categories (StringVec).
+    Returns the names of all tech categories (StringVec).
```

Comment:
It would be nice o to add information about arguments to C++ documentation.

Reply:
there are none
https://github.com/freeorion/freeorion/blob/master/universe/Tech.h#L195
https://github.com/freeorion/freeorion/blob/master/python/EmpireWrapper.cpp#L470